chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -846,13 +850,13 @@ sync.error.emptyResponseServer 		= Empty response from server.
 sync.error.invalidCharsFilename		= The filename '%S' contains invalid characters.\n\nRename the file and try again. If you rename the file via the OS, you will need to relink it in Zotero.
 sync.error.apiKeyInvalid			= %S could not authenticate your account. Please re-enter your account details.
 
-sync.lastSyncWithDifferentAccount	= This Zotero database was last synced with a different %1$S account (‘%2$S’) from the current one (‘%3$S’).
-sync.localDataWillBeCombined		= If you continue, local data will be combined with data from the ‘%1$S’ account on %2$S.
-sync.localGroupsWillBeRemoved1		= Local groups, including any with changed items, will also be removed from this computer.
-sync.avoidCombiningData				= To avoid combining data, revert to the ‘%S’ account or use the Reset options in the Sync pane of the Zotero preferences.
-sync.unlinkWarning					= Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally.
-sync.warning.emptyLibrary			= You are about to sync the ‘%1$S’ account to an empty %2$S database. This could happen if you removed your previous database or if the location of your %2$S data directory changed.
-sync.warning.existingDataElsewhere  = If your %S data exists elsewhere on your computer, you should move it to your current data directory or change your data directory to point to the existing data.
+account.unlinkWarning					= Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally.
+account.warning.emptyLibrary			= You are about to sync the ‘%1$S’ account to an empty %2$S database. This could happen if you removed your previous database or if the location of your %2$S data directory changed.
+account.warning.existingDataElsewhere	= If your %S data exists elsewhere on your computer, you should move it to your current data directory or change your data directory to point to the existing data.
+account.lastSyncWithDifferentAccount	= This Zotero database was last synced with a different %1$S account (‘%2$S’) from the current one (‘%3$S’). If you continue, the data and files belonging to ‘%2$S’ will be removed from your computer and Zotero will restart.
+account.alternativePickNewDataDir		= Alternatively you can click ‘%S’ to choose a new data directory for ‘%S’.
+account.confirmLastUsername				= To confirm that you want to remove all data and files associated with ‘%1$S’ enter ‘%1$S’ in the textbox below.
```

Comment:
> GitHub uses repository name for repo deletion, hence the decision, but we can change that to 'delete' I guess.

Yeah, but I think that GitHub example is a bit different — you can have multiple repos, and that's confirming that you're deleting the right one. In this case you're pretty much just confirming that you want to go ahead with this action, and I think entering your username for that is a little weird. I don't love "delete", but I don't have a great alternative other than "delete %1$S data", which seems too error-prone.